daffodil-core/src/main/scala/org/apache/daffodil/compiler/Compiler.scala

```diff
@@ -64,16 +63,6 @@ object BothParserAndUnparser extends ParserOrUnparser
 
 final class ProcessorFactory private(
   private var optRootSpec: Option[RootSpec],
```

Comment:
Can we remove this `var`, similar to what we did with the `DataProcessor`?

Reply:
Remove it completely or change it to `val`?

Reply:
Yeah, change to a val. My mistake, that wasn't very clear.